make/common/NativeCompilation.gmk

```diff
@@ -438,7 +438,8 @@ define SetupCompileNativeFileBody
             # For assembler calls, no need to build dependency list.
 	    $$(call ExecuteWithLog, $$@, $$(call MakeCommandRelative, \
 	        $$($1_COMPILER) $$($1_FLAGS) \
-	          $(CC_OUT_OPTION)$$($1_OBJ) /Ta $$($1_SRC_FILE)))
+	          $(CC_OUT_OPTION)$$($1_OBJ) /nologo /c /Ta $$($1_SRC_FILE) 2>&1 \
+	            | $(TR) -d '\r' | $(GREP) -v -e "Assembling:" || test "$$$$?" = "1" ;))
```

Comment:
Please use the `-nologo -c -Ta` format for the options.

Reply:
done